content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/create-slack-alerts/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Create Slack Alerts from Enforce Cloud Events"
+type: "article"
+description: "Creating Slack Alerts from Enforce Cloud Events"
+date: 2023-04-11T15:22:20+01:00
+lastmod: 2023-04-11T15:22:20+01:00
+draft: false
+tags: ["Enforce", "Product", "Cloudevents"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 650
+toc: true
+---
+
+> _This documentation is related to Chainguard Enforce. You can request access to the product by selecting **Chainguard Enforce** on the [inquiry form](https://www.chainguard.dev/contact?utm_source=docs)._
+
+Chainguard Enforce for Kubernetes can send Cloudevents as webhook requests that you can subscribe to and use to do things like generate alerts, create Github issues, post messages to Slack channels and more. This tutorial is meant to serve as a companion to the [Enforce Events slack notifier](https://github.com/chainguard-dev/enforce-events/tree/main/github-issue-opener) example application. It will guide you through creating and deploying a Google Cloud Run service that will send a Slack message to a specifed channel for any Enforce policy violation or admission event.
+
+You can also opt to run the example application elsewhere. As long as it is publicly accessible to the [Enforce CIDR network ranges](/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/network-requirements/#cidr-ranges), you will be able to receive Cloudevents and create Slack messages with the example application. If you opt to run the demo application outside of Cloud Run, you can skip to the [Subscribing to Enforce Cloudevents](#subscribing-to-enforce-cloudevents) section, provided you know the publicly accessible URL to your deployed instance of the demo.
```

Comment:
```suggestion
You can also opt to run the example application elsewhere. As long as it is publicly accessible to the [Enforce CIDR network ranges](/chainguard/chainguard-enforce/reference/network-requirements/#cidr-ranges), you will be able to receive Cloudevents and create Slack messages with the example application. If you opt to run the demo application outside of Cloud Run, you can skip to the [Subscribing to Enforce Cloudevents](#subscribing-to-enforce-cloudevents) section, provided you know the publicly accessible URL to your deployed instance of the demo.
```

And another general reminder to double check that all the URLs are relative and updated!